### ISSUE TITLE
Contacts sort options

### ISSUE THREAD

**Issue by sapphirepro** [telegramdesktop/tdesktop#26901]
### Is your feature request related to a problem?

It's hard to review contacts on Telegram, because it's only sorted by last online state. Scrolling them and then suddenly someone coming online messes all up + it's very hard to find specific contacts.

### Describe the solution you'd like

Add sort options, please. At minimum required is search by alphabet. A-Z etc, then can scroll all contacts by alphabet ignoring their last online state. It would make it much more convenient.

Thank you a lot in advance!

### Describe alternatives you've considered

Add sort options, please. At minimum required is search by alphabet. A-Z etc, then can scroll all contacts by alphabet ignoring their last online state. It would make it much more convenient.

Thank you a lot in advance!

### Additional context

We are talking of sidebar menu => Contacts.

**Comment by john-preston** [comment]
@sapphirepro There is a button in the top right box corner.

**Comment by sapphirepro** [comment]
> @sapphirepro There is a button in the top right box corner.

It is NOT working as expected, cause regardless of alphabet it still keeps online/offline as priority. If alphabetical order online/offline status must be fully ignored. So feature is either missing or buggy, depending of what you preffer

**Comment by sapphirepro** [comment]
Yes. Just checked. It's a bug. In Android app sorts ok. In Linux Desktop app shows first online and few sorted by recent online, then alphabetically, so algorithm is broken or contains bug

**Comment by john-preston** [comment]
@sapphirepro I cannot reproduce that :(

**Comment by sapphirepro** [comment]
Look. Both sorting modes. Are any of them alphabetical from top to bottom? NO!

![Screenshot_20231007_065601-1](https://github.com/telegramdesktop/tdesktop/assets/35900229/2fd6aeb1-72b2-4e3a-8f9a-7ef3dbf684e6)
![Screenshot_20231007_065614](https://github.com/telegramdesktop/tdesktop/assets/35900229/826d6981-b662-4517-a375-85537e3475e8)

**Comment by sapphirepro** [comment]
> Look. Both sorting modes. Are any of them alphabetical from top to bottom? NO!
> 
> ![Screenshot_20231007_065601-1](https://user-images.githubusercontent.com/35900229/273364079-2fd6aeb1-72b2-4e3a-8f9a-7ef3dbf684e6.png) ![Screenshot_20231007_065614](https://user-images.githubusercontent.com/35900229/273364080-826d6981-b662-4517-a375-85537e3475e8.png)

You really can not see that on first screenshot that should be by Alphabet top rows still count with last online time?

**Comment by john-preston** [comment]
It looks like switching didn't work at all 🤔

**Comment by john-preston** [comment]
Mayne that's a locale issue, because for sorting it uses some stripped from diacritics etc strings.

**Comment by sapphirepro** [comment]
> Maybe that's a locale issue, because for sorting it uses some stripped from diacritics etc strings.

Possible. Worth investigating I guess. In mobile app all works normally as expected

**Comment by sapphirepro** [comment]
> It looks like switching didn't work at all 🤔

Well, it changes stuff (as you can see lists are not equal), but does it some very weird way
